vep/__init__.py

```diff
@@ -6,6 +6,11 @@
 import sys
 
 
+# pass-through function to enable line-wise output from commands called via sh
+def print_line(line):
+    print(line)
```

Comment:
This kinda makes my brain hurt, so I'd really like to hear what made you write this code :-)